common/src/main/java/org/apache/cassandra/sidecar/common/ICassandraAdapter.java

```diff
@@ -28,6 +28,14 @@
  */
 public interface ICassandraAdapter
 {
+    /**
+     * @return a list of {@link NodeStatus} for the Cassandra cluster
+     */
```

Comment:
TBH, I do not know why the return type is a list. The `ICassandraAdapter` is to interact with the local Cassandra node that pairs with the sidecar. Why returning the list of node status? Sidecar does not support bulk operation yet, and it is not even implemented. 
Let not add the comment for the method. The comment might be no accurate. We might want to refactor the method later since it does not make too much sense.